app/controllers/catalog_controller.rb

```diff
@@ -80,21 +85,22 @@ def guest_uid_authentication_key(key)
     #  (useful when user clicks "more" on a large facet and wants to navigate alphabetically across a large set of results)
     # :index_range can be an array or range of prefixes that will be used to create the navigation (note: It is case sensitive when searching values)
 
-    config.add_facet_field 'format', label: 'Format'
-    config.add_facet_field 'pub_date_ssim', label: 'Publication Year', single: true
-    config.add_facet_field 'subject_ssim', label: 'Topic', limit: 20, index_range: 'A'..'Z'
-    config.add_facet_field 'language_ssim', label: 'Language', limit: true
-    config.add_facet_field 'lc_1letter_ssim', label: 'Call Number'
-    config.add_facet_field 'subject_geo_ssim', label: 'Region'
-    config.add_facet_field 'subject_era_ssim', label: 'Era'
-
-    config.add_facet_field 'example_pivot_field', label: 'Pivot Field', pivot: ['format', 'language_ssim']
-
-    config.add_facet_field 'example_query_facet_field', label: 'Publish Date', query: {
-      years_5: { label: 'within 5 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 5} TO *]" },
-      years_10: { label: 'within 10 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 10} TO *]" },
-      years_25: { label: 'within 25 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 25} TO *]" }
-    }
+    config.add_facet_field 'marc_resource', label: 'Access', limit: 5
+    config.add_facet_field 'format', label: 'Resource Type', limit: 5
+    config.add_facet_field 'language_facet', label: 'Language', limit: 5
+    config.add_facet_field 'author_display', label: 'Author/Creator', limit: 5
+    config.add_facet_field 'subject_topic_facet', label: 'Subject', limit: 5
+    config.add_facet_field 'title_series_t', label: 'Collection', limit: 5
+    config.add_facet_field 'subject_geo_facet', label: 'Region', limit: 5
+    config.add_facet_field 'subject_era_facet', label: 'Era', limit: 5
+
+    # config.add_facet_field 'example_pivot_field', label: 'Pivot Field', pivot: ['format', 'language_ssim']
+
+    # config.add_facet_field 'example_query_facet_field', label: 'Publish Date', query: {
+    #   years_5: { label: 'within 5 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 5} TO *]" },
+    #   years_10: { label: 'within 10 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 10} TO *]" },
+    #   years_25: { label: 'within 25 Years', fq: "pub_date_ssim:[#{Time.zone.now.year - 25} TO *]" }
+    # }
```

Comment:
can we remove these comments?

Reply:
Sure. I'll remove them from #114 and rebase this one.

Reply:
That is done.